### PR TITLE
Feature/tokenregistry admin accounts

### PR DESCRIPTION
**Pull request by Jedi2002**

### CODE REVIEW

File: x/tokenregistry/types/genesis.go [hunk 1]
```diff
@@ -30,7 +37,38 @@ func GetGenesisStateFromAppState(marshaler codec.JSONCodec, appState map[string]
...
+				AdminAddress: "sif1l7hypmqk2yc334vc6vmdwzp5sdefygj2ad93p5",
+			},
+			{
+				AdminType:    AdminType_ETHBRIDGE,
+				AdminAddress: "sif1syavy2npfyt9tcncdtsdzf7kny9lh777yqc2nd",
+			},
+			{
+				AdminType:    AdminType_TOKENREGISTRY,
+				AdminAddress: "sif1syavy2npfyt9tcncdtsdzf7kny9lh777yqc2nd",
+			},
```
Comment (AustinoBombino) [inline x/tokenregistry/types/genesis.go:66]: @Jedi2002 is this sif1syavy2npfyt9tcncdtsdzf7kny9lh777yqc2nd the real admin account? or do we still need to update the migrator?

Comment (Jedi2002) [reply x/tokenregistry/types/genesis.go:66]: WE need to add to the migrator as well , I was thinking of an if condition like so 
if ctx.ChainID() != "sifchain-1" { } 
if ctx.ChainID() != "sifchain-devnet-1" { } 
if ctx.ChainID() != "sifchain-testnet-1" { } 

Just need the correct addresses for that

File: x/tokenregistry/keeper/genesis.go [hunk 2]
```diff
@@ -8,23 +8,19 @@ import (
...
-			panic(err)
+	if state.AdminAccounts != nil {
+
+		for _, adminAccount := range state.AdminAccounts.AdminAccounts {
+			k.SetAdminAccount(ctx, adminAccount)
 		}
-		k.SetAdminAccount(ctx, addr)
-	}
-	if state.Registry != nil {
-		k.SetRegistry(ctx, *state.Registry)
```
Comment (canercandan) [inline x/tokenregistry/keeper/genesis.go:19]: @Jedi2002 I think those 2 lines have been mistakenly removed

**Review by canercandan — APPROVED** [review]
LGTM

**Pull request merged**